src/data/markdown/docs/01 guides/03 Results visualization/CSV.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'CSV'
+excerpt: ''
+---
+
+You can also make k6 output detailed statistics in a CSV format by using the `--out/-o` option for `k6 run`, like this:
+
+<div class="code-group" data-props='{"labels": ["CLI"]}'>
+
+```shell
+$ k6 run --out csv=my_test_result.csv script.js
+```
+
+</div>
+
+Or if you want to get the result gzipped, like this:
+
+<div class="code-group" data-props='{"labels": ["CLI"]}'>
+
+```shell
+$ k6 run --out csv=my_test_result.gz script.js
+```
+
+</div>
```

Comment:
This,technically, will be available after 0.28.0 is released, maybe we can add this as a note somewhere

Reply:
Yeah, maybe we could comment it out for now, and uncomment it once 0.28.0 is released.

Reply:
Do you want me to comment the entire document? Or should I leave a note?

Reply:
Sorry ... I have apparently forgotten to answer you are haven't seen the email. 
This although makes this easier as ... well we released 0.28.0 :D